influxql/token.go

```diff
@@ -54,8 +55,8 @@ const (
 	SEMICOLON // ;
 	DOT       // .
 
-	keyword_beg
-	// Keywords
+	keywordBeg
+	// ALL and other Keywords
```

Comment:
Sure, I see what you're saying. Sure, go ahead and leave the comments in. I don't think they're hurting anything. :)